get-entitlements-from-dvla/src/entitlements.js

```diff
@@ -0,0 +1,36 @@
+const validDrivingLicenceNumbers = require('./entitlements.json');
+const crypto = require('crypto');
+
+class Entitlements {
+
+	/**
```

Comment:
Can you put a summary of function to the comments here, rather than just the input and output variables?